matminer/featurizers/function.py

```diff
@@ -73,15 +75,53 @@ def exp_dict(self):
                                                    self.combo_function))
              for n in range(1, self.multi_feature_depth+1)])
 
+    def featurize(self, *args):
+        """
+        Main featurizer function, essentially iterates over all
+        of the functions in self.function_list to generate
+        features for each argument.
+
+        Args:
+            *args: list of numbers to generate functional output
+                features
+
+        Returns:
+            list of functional outputs corresponding to input args
+        """
+        return list(self._exp_iter(*args, postprocess=self.postprocess))
+
+    def feature_labels(self):
+        """
+        Returns:
+            Set of feature labels corresponding to expressions
+        """
+        return self._feature_labels
```

Comment:
Should we add in a "not fitted" error here?

Reply:
Added, tested.